.travis.yml

```diff
@@ -7,7 +7,7 @@ jobs:
       script: black . --check --diff
     - stage: check
       language: javascript
-      install: cd frontend && npm install
+      install: cd frontend && nvm install node && npm install
```

Comment:
You can use the `node_js` attribute in `.travis.yml` to specify the node version: https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions

Reply:
Updated!